rune/libenclave/internal/runtime/pal/skeleton/running_skeleton_with_tls_server.md

```diff
@@ -71,6 +66,42 @@ Then build the skeleton docker image with the command:
 docker build . -t skeleton-enclave
 ```
 
+### Build DCAP-based remote attestation skeleton docker image
+
+Assume your host is CentOS.
+
+Type the following commands to create a Dockerfile:
+
+```Shell
+cp /etc/pki/ca-trust/extracted/openssl/ca-bundle.trust.crt ./
+cp ${path_to_inclavare_containers}/ra-tls/build/bin/Wolfssl_Enclave.signed.so ./
+cp /etc/sgx_default_qcnl.conf ./
+cp /usr/lib64/libsgx_pce.signed.so ./
+cp /usr/lib64/libsgx_qe3.signed.so ./
+
+cat >Dockerfile <<EOF
+FROM centos:8.1.1911
+
+WORKDIR /
+
+COPY Wolfssl_Enclave.signed.so /
+
+RUN mkdir -p /etc/ssl/certs/
+COPY ca-bundle.trust.crt /etc/ssl/certs/
+
+COPY sgx_default_qcnl.conf /etc/
+
+COPY libsgx_pce.signed.so /usr/lib64
+COPY libsgx_qe3.signed.so /usr/lib64
```

Comment:
This assumes PCE and QE3 both exist in container. It is OK but not the goal. Currently intelsgx has implemented protobuf message used to communicate with aesmd, so it is worthy doing better here in future. I mean PCE and QE3 are hosted by aesmd.

Reply:
Please submit an issue to address the above issue and set its fix milestone to 0.7.0